tests/autobahn/Dockerfile.aiohttp

```diff
@@ -0,0 +1,13 @@
+FROM python:3.9.1
+
+ENV DEBIAN_FRONTEND=noninteractive
+RUN \
+    apt-get update && \
+    apt-get -y install --no-install-recommends wait-for-it && \
+    apt-get clean && \
+    rm -rf /var/lib/apt/lists/*
+
+ADD ./ /src
+ENV AIOHTTP_NO_EXTENSIONS=1
+WORKDIR /src
+RUN python setup.py install
```

Comment:
<details>
<summary><strong>NOOOOOOOO!!111</strong></summary>

![NOOOOOOOO!!111](https://img.17qq.com/images/hmmmlknkklv.jpeg)
</details>

Invoking `setup.py` directly is a very bad idea™. Pip should be used instead.